sale_layout_category_hide_detail/views/sale_order_report_templates.xml

```diff
@@ -52,7 +52,7 @@
             expr="//tbody[hasclass('sale_tbody')]//tr[hasclass('is-subtotal', 'text-right')]/td/strong"
             position="after"
         >
-      <strong class="mr16">Subtotal <span t-field="current_section.name" /></strong>
+      <strong class="mr16">Subtotal</strong>
```

Comment:
This is ripping a feature that our client needs, so please restore it

Reply:
Ok, commit removed.